compliance-monitor/templates/details.md.j2

```diff
@@ -1,29 +1,32 @@
 {% for subject, subject_result in results.items() -%}
 # {{ subject }}
-
 {% for scopeuuid, scope_result in subject_result.items() -%}
-
 {% if not scope_result.relevant -%}
 ## {{ scope_result.name }}
+
 No recent test results available.
+
 {% endif -%}
 {% for version in scope_result.relevant -%}
-{% set version_result = scope_result.versions[version] -%}
+{%- set version_result = scope_result.versions[version] -%}
 ## {{ scope_result.name }} {{ version }} ({{ version_result.validity }}): {{ version_result.result | verdict }}
-
 {% for target, target_result in version_result.targets.items() -%}
 ### Target {{ target }}: {{ target_result.result | verdict }}
 
-|---|---|---|
 | testcase id | result | description |
 |---|---|---|
 {% for testcase_id in target_result.testcases -%}
 {% set testcase = version_result.testcases[testcase_id] -%}
-{% set result = version_result.results[testcase_id].result if testcase_id in version_result.results else 0 -%}
-| {{ testcase.id }} | {{ result | verdict }} | {{ testcase.description | trim }} |
-{% endfor -%}
-|---|---|---|
-
+{% set res = version_result.results[testcase_id] if testcase_id in version_result.results else dict(result=0) -%}
```

Comment:
I'm not very familiar with this table, but shouldn't this be:
```suggestion
{% set res = version_result.results[testcase_id].result if testcase_id in version_result.results else dict(result=0) -%}
```

Reply:
No, it's correct the way it's now. See the else branch

Reply:
(I did test it!)